Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,582 @@
+
+You can install the `Microsoft.OData.ModelBuilder` package as follows:
+
+# [Visual Studio](#tab/visual-studio)
+
+In the Visual Studio **Package Manager Console**:
+
+```powershell
+Install-Package Microsoft.OData.ModelBuilder
+```
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+```dotnetcli
+dotnet add package Microsoft.OData.ModelBuilder
+```
+
+---
+
+## EDM model
+
+The EDM model refers to the in-memory representation of the OData schema. It defines the data types and endpoints exposed by the OData service. This allows clients to consume the API in a predictable, type-safe manner. The EDM model is represented by the [`IEdmModel`](/dotnet/api/microsoft.odata.edm.iedmmodel) interface defined in the `Microsoft.OData.Edm` package. When you create an OData service, you need specify the `IEdmModel` that describes the service. This interface exposes methods for retrieving the data types, metadata and other information defined in the schema. You specify the EDM model by passing an instance of `IEdmModel` to the `ODataOptions.AddRouteComponents` method:
+
+```c#
+services.AddControllers().AddOData(options => options.AddRouteComponents(edmModel));
+```
+
+There are a number of ways to create an `IEdmModel` instance:
+- You can use the `CsdlReader` class to parse an `IEdmModel` from a stream that contains the OData schema in [CSDL XML](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html) or [CSDL JSON](https://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html) format. The `CsdlReader` class is defined in the `Microsoft.OData.Edm` package. To learn more, [visit this article](/odata/odatalib/edm/read-write-model).
+- You can manually construct a model by creating an instance of the [`EdmModel`](/dotnet/api/microsoft.odata.edm.edmmodel) class (which implements `IEdmModel`). The `EdmModel` class is also part of the `Microsoft.OData.Edm` package. To learn more about creating EDM models manually, visit the [EDM documentation](/odata/odatalib/edm/build-basic-model).
+- You can use the `Microsoft.OData.ModelBuilder` package to easily create an `IEdmModel` based on CLR classes. The package provides 2 main classes:
+    - `ODataConventionModelBuilder` used to quickly create models from CLR classes based on opinionated conventions (for example, it can infer key properties based on conventional names like `Id`). This is the easiest method and what we use in most of the samples in this documentation. For example usage, [visit this article](/odata/webapi/convention-model-builder).
+    - `ODataModelBuilder` gives you more control to specify things explicitly but requires more work to use. For example usage, [visit this article](/odata/webapi/model-builder-nonconvention).
```

Comment:
I think because you're talking about the 2 main classes, this sentence should probably start with `- `ODataModelBuilder` that gives you...`.

Reply:
I've rephrased it as "`ODataModelBuilder`, that gives you..."